### PR TITLE
Added simple Agenda page to Contribute tab

### PR DESCRIPTION
**Pull request by Bangalisch**
A start as per #1081 raised.

Very simple page that has the calender easy to find.

There's no nice looking buttons or anything. Maybe someone can make this look better?

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -266,7 +266,8 @@ const sidebarContribute = [
       },
       ['/Contribute/Design', 'Design'],
       ['/Contribute/Translate', 'Translate'],
-      ['/Contribute/Misc', 'Miscellaneous']
+      ['/Contribute/Misc', 'Miscellaneous'],
+      ['/Contribute/Agenda', 'Calls Calender']
```
Comment (pavlenex) [inline docs/.vuepress/config.js:270]: It's calendar! 😄 
```suggestion
      ['/Contribute/Agenda', 'Calls Calendar']
```

Comment (pavlenex) [reply docs/.vuepress/config.js:270]: Should we call these office hours, open calls? Calls calendar doesn't really give context on what's going on?

File: docs/Contribute/Agenda.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Calls Calender.
+
+All calls, design, development, and Documentation (marketing, SEO) are open to everyone to join.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:3]:
```suggestion
All calls, design, development, and documentation (marketing, SEO) are open to everyone to join and participate or listen. 
```

File: docs/Contribute/Agenda.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Calls Calender.
+
+All calls, design, development, and Documentation (marketing, SEO) are open to everyone to join. 
+If you want to see the upcoming scheduled Calls, follow the guide below.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:4]:
```suggestion
If you want to see the upcoming scheduled calls, follow the guide below. 
```

File: docs/Contribute/Agenda.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Calls Calender.
+
+All calls, design, development, and Documentation (marketing, SEO) are open to everyone to join. 
+If you want to see the upcoming scheduled Calls, follow the guide below. 
+
+Calls are scheduled via Github issues.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:6]: Scheduling calls is irrelevant for this page. A user landing on this page should know how to join, and you should try to proiritize that ,scheduling is always done by people who already know the process, if you really want to document it, try positioning it after the "subscribing to calls"

File: docs/Contribute/Agenda.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Calls are scheduled via Github issues. 
+You can find all upcoming calls with their designated tags [Here](https://github.com/btcpayserver/organization/issues)
+
+There are two requirements for an issue to be included:
+
+* The issue needs to include a UTCTime meta property following this format: UTCTime: 2020-10-14 9:00 UTC -7
+* A maintainer needs to add the `Call` label to the issue (this prevents spam)
+* The calendar auto-updates whenever a new issue is created or an existing issue is edited
+
+## BTCPay Server Organization Calender.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:15]:
```suggestion
## BTCPay Server Calls Calendar
```

**Review by pavlenex — COMMENTED** [review]
A few nits, but overall good page!

File: docs/.vuepress/config.js (hunk 1 above)
Comment (Bangalisch) [reply docs/.vuepress/config.js:270]: What ya say about "Open office hours" ?

Comment (pavlenex) [reply docs/.vuepress/config.js]: That may imply there's an office or even working hours :) How about "Open Community Calls"?

Comment (Bangalisch) [reply docs/.vuepress/config.js]: Allright will do :)

File: docs/Contribute/Agenda.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+# Open Community Calls
+
+We welcome everyone for our Open office hours.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:3]:
```suggestion
BTCPay Server operates as an open-organization. All of our meetings where we discuss the development, design, documentation or other topics are opened for anyone to join in and participate. To get involved, you can subscribe to our open-calendar and stay in the loop on what we're working on, learn, ask questions or just join us in building open-source bitcoin payment rails.
```

File: docs/Contribute/Agenda.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+# Open Community Calls
+
+We welcome everyone for our Open office hours. 
+Here we talk through the work ahead, upcoming and previous releases, bug fixes, Design decissions and the nitty gritty on Documents and Marketing. 
+Interest in joinen? DO IT! You don't have to participate, you're welcome to just listen and enjoy the banter.
```
Comment (pavlenex) [inline docs/Contribute/Agenda.md:5]: I suggest we keep things concise 😄 
```suggestion
```

**Review by pavlenex — COMMENTED** [review]
Just a final re-phrasing, made things a bit more concise in a single paragraph, but this is now good to go, thanks for clearning things up @Bangalisch you rock 🎸

**Comment by pavlenex** [comment]
@Bangalisch can you please add a link to this page, I think we should def link the calls page to our contribute page.

Screenshot 2022-05-10 at 09 44 18

**Comment by Bangalisch** [comment]
@pavlenex Also added to the list above that sentence te reflect the menu on the left.

**Pull request merged**